src/NServiceBus.Core/MessageDeserializationException.cs

```diff
@@ -22,9 +23,9 @@ public MessageDeserializationException(string message) : base(message)
         /// Initializes a new instance of <see cref="MessageDeserializationException" />.
         /// </summary>
         /// <param name="innerException"> The exception that is the cause of the current exception.</param>
-        /// <param name="transportMessageId">The id of the <see cref="TransportMessage" /> that failed to deserialize.</param>
-        public MessageDeserializationException(string transportMessageId, Exception innerException)
-            : base("An error occurred while attempting to extract logical messages from transport message " + transportMessageId, innerException)
+        /// <param name="incomingPhysicalMessageId">The id of the <see cref="IncomingMessage" /> that failed to deserialize.</param>
+        public MessageDeserializationException(string incomingPhysicalMessageId, Exception innerException)
```

Comment:
Is there any reason to differentiate between a message id and an `incomingPhysicalMessageId`? What I find more interesting is, that the previous version named this parameter `transport message id` but it seems to consider this param as the regular message id, not the transport specific internal message id if I understand this correctly. cc @andreasohlund